odl/operator/default_ops.py

```diff
@@ -103,6 +104,25 @@ def inverse(self):
     def adjoint(self):
         """Adjoint, given as scaling with the conjugate of the scalar.
 
+        Examples
+        --------
+        In the real case, the adjoint is the same as the operator
+
+        >>> r3 = odl.rn(3)
+        >>> x = r3.element([1, 2, 3])
+        >>> op = ScalingOperator(r3, 2)
+        >>> op.adjoint(x) == op(x)
```

Comment:
Don't rely on this behavior of `==`, it will change.

Reply:
How should I compare, `all_almost_equal`? But then it becomes more unit-test-like...

Reply:
If it doesn't get too busy, just do
```python
>>> op(x)
(output)
>>> op.adjoint(x)  # the same
(output)
```